gradle.properties

```diff
@@ -1,3 +1,4 @@
 org.gradle.caching=true
 group=org.ballerinalang.microsoft.teams
-version=2.1.0
+version=2.2.0
+ballerinaLangVersion=2201.0.0
```

Comment:
Shall we remove this `ballerinaLangVersion`?

We purposefully removed it earlier.

if we have java-wrappers only, we need to include this.

Reply:
Yeah. But without this we cannot build the code. 
```
> Could not get unknown property 'ballerinaLangVersion' for root project 'module-ballerinax-microsoft.teams' of type org.gradle.api.Project.
```

Reply:
Removed as suggested.

Reply:
Need to remove the usage of `ballerinaLangVersion` also in the project.